### PR TITLE
Added space above error message.

### PR DESCRIPTION
**Pull request by suryakant-krish**
### Description (*)

1. This is direct PR because it  is small change.
2. Area Frontend. when user group product add to cart without add quantity error message show. 

### Fixed Issues (if relevant)

1. This is direct PR.
2. Go to frontend > Find any group product **Ex** "Set of Sprite Yoga Straps".  > on product page click on add to cart button error message show.
3. Reproduce on Magento 2.3

### Manual testing scenarios (*)

1. Actual Result.
actual-result

2. Expected Result.
expected-result

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-4181](https://jira.corp.magento.com/browse/ENGCOM-4181) has been created to process this Pull Request

**Comment by Karlasa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [new Magento instance](https://pr-21074.instances.magento-community.engineering).
Admin access: [https://pr-21074.instances.magento-community.engineering/admin](https://pr-21074.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by Karlasa — CHANGES_REQUESTED** [review]
@suryakant-krish @dmytro-ch 
This fix looks bit too general and will affect a lot of other places that has this class. 
maybe add margin into #validation-message-box and under grouped product module?

**Comment by dmytro-ch** [comment]
@Karlasa, thank you for the update.

I was also thinking about that but decided that having kind of base `10px` space above this block allows us to make sure there are no additional issues with other blocks inserted before the `product.info.addtocart` block.
Also, there are other rules overriding this value for other product types, e.g. `20px` for configurable products etc.

But yes, I agree that it makes sense to make the changes for grouped products only for this particular case in order to avoid some unexpected behavior. 

Thank you!

**Comment by suryakant-krish** [comment]
@dmytro-ch and @Karlasa

I have updated code only for group product used page layout handle hierarchy.

Please review this and let me know.

**Comment by sivaschenko** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, here is your [new Magento instance](https://pr-21074.instances.magento-community.engineering).
Admin access: [https://pr-21074.instances.magento-community.engineering/admin](https://pr-21074.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less [hunk 1]
```diff
@@ -293,6 +293,12 @@
         }
     }
 
+    .page-product-grouped {
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less]: Grouped product specific code should be placed under right module Magento__GroupedProduct in this case.

**Review by Karlasa — CHANGES_REQUESTED** [review]

**Comment by suryakant-krish** [comment]
@Karlasa,

I have moved code under "Magento__GroupedProduct".

File: app/design/frontend/Magento/luma/Magento_GroupedProduct/web/css/source/_module.less [hunk 2]
```diff
@@ -70,6 +70,9 @@
                 clear: left;
             }
         }
+        .box-tocart {
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_GroupedProduct/web/css/source/_module.less:74]: add extra new line above it to fit with coding standards :)

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less [hunk 3]
```diff
@@ -293,6 +293,8 @@
         }
     }
 
+
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less]: remove these 2 added lines :)

**Comment by Karlasa** [comment]
@suryakant-krish 
Thanks, just a bit more code clean up and I'm happy 👍

**Comment by suryakant-krish** [comment]
@Karlasa,

I have updated code as you have mentioned.

**Review by Karlasa — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, thank you for the review.
[ENGCOM-4181](https://jira.corp.magento.com/browse/ENGCOM-4181) has been created to process this Pull Request

**Comment by VasylShvorak** [comment]
✔️ QA passed

**Pull request merged**